docs/detections-as-code.md

```diff
@@ -0,0 +1,57 @@
+# Detection as Code (DaC) Components in Detection-Rules Repo
+
+The **detection-rules** repository contains features for **Detections as Code (DaC)**. These components, including CLI options and workflows, provide methods to help apply DaC principles in practice. The specific DaC architecture should be carefully considered before implementation, for more conceptual ideas and details about DaC, refer to the [DaC Documentation](https://dac-reference.readthedocs.io/en/latest/). Reference implementation is shared to facilitate experimentation and community contributions.
+
+> [!NOTE]
+> This guidance outlines the support scope and best practices for using DaC components within the detection-rules repo. Users should take full responsibility for their usage of this repo, thoroughly test these tools in their environments, and verify functionality before using them.
+
+---
+
+## Support and Scope
+
+Supported DaC components that interact with the Elastic Security Solution:
+- kibana export-rules ([link](https://github.com/elastic/detection-rules/blob/main/CLI.md#exporting-rules))
+- kibana import-rules ([link](https://github.com/elastic/detection-rules/blob/main/CLI.md#using-import-rules))
+- import-rules-to-repo ([link](https://github.com/elastic/detection-rules/blob/main/CLI.md#import-rules-to-repo))
+- export-rules-from-repo ([link](https://github.com/elastic/detection-rules/blob/main/CLI.md#uploading-rules-to-kibana))
+
+We welcome general questions, feature requests, and bug reports through the following channels:
+- **GitHub Issues**: For raising general questions, bugs, and feature requests related to the detection-rules repo.  
+  [GitHub Issues](https://github.com/elastic/detection-rules/issues)
+- **Community Slack**: For informal discussions or questions (note that message history is limited to 30 days).  
+  [Elastic Security Community Slack](https://elasticstack.slack.com/archives/C06TE19EP09)
+
+Support tickets related to this DaC reference implementation can be opened with Elastic, however since the logic is just a wrapper of the underlying product API's, we will not be able to treat DaC related support requests as a Severity-1 (Immedediate time frame). Urgent DaC issues can be resolved via direct interaction with the underlying Kibana API or Elastic Security Solution UI. We will ask customers to resort to these methods to resolve downtime or urgent issues.
```

Comment:
```suggestion
Support tickets related to this DaC reference implementation can be opened with Elastic, however since the logic is just a wrapper of the underlying product API's, we ask to resolve urgent DaC issues via direct interaction with the underlying [Kibana APIs](https://www.elastic.co/docs/api/doc/kibana/v8/group/endpoint-security-detections-api) or [Elastic Security Solution UI](https://www.elastic.co/guide/en/security/current/rules-ui-management.html), as we will not be able to treat DaC related support requests as a high severity (immediate time frame).
```